AppLovin MAX Demo App - ObjC/AppLovin MAX Demo App - ObjC/AppLovin/MRECs/Programmatic/ALDemoProgrammaticMRECViewController.m

```diff
@@ -16,6 +16,7 @@ @interface ALDemoProgrammaticMRECViewController()<ALAdLoadDelegate, ALAdDisplayD
 @implementation ALDemoProgrammaticMRECViewController
 static const CGFloat kMRECHeight = 250.0f;
 static const CGFloat kMRECWidth = 300.0f;
+static const CGFloat kcallBackTableviewHeight = 325.0f;
```

Comment:
Casing:

```suggestion
static const CGFloat kCallBackTableViewHeight = 325.0f;
```

Also, is it always 325 tall? Where did that value come from? How would this look on an iPad?

Reply:
In the IB implementation there's a height constraint on the tableview, so I brought it into the programatic implementation.

Reply:
Also, instead of hardcoding a value here and in SB (which may easily lead to discrepancies) - I suggest referencing the specific height constraint in the storyboard and getting the height from the constant property of that constraint.

Reply:
decided to remove the height constraint completely on both ib and programmatic implementations, allowing it to grow up to the the top of the ad. The height of the ad is constrained.

Reply:
Cool - I think that's the dynamic way to do it :)